bigtable-hbase-1.x-parent/bigtable-hbase-1.x-integration-tests/pom.xml

```diff
@@ -86,6 +86,70 @@ limitations under the License.
         </plugins>
       </build>
     </profile>
+    <profile>
```

Comment:
Do we need a new profile for this?

Can't we execute:

mvn -P bigtableIntegrationTest -Dbigtable.version=LATEST?

Reply:
Interestingly, `LATEST` from the command line doesn't seem to work for me (but hardcoding a snapshot version, ie 1.12.3-SNAPSHOT, does). Either way, we'd still have to add the `<repositories>` section if we reused the main profile, which I'm not sure I'd prefer to do.

Reply:
In that case can we make the profile additive?

mvn -P bigtableIntegrationTest -PuseVeneerSnapshot?

Reply:
Done. Also moved the profile to the parent to activate for all unit tests (not just integration modules)